content/kubeone/master/guides/addons/_index.en.md

```diff
@@ -53,6 +53,10 @@ data:
 
 **Note:** The `b64enc` function is a [`sprig` function][sprig-b64enc].
 
+## Addons API Reference
+
+Take a look at the [KubeOne's autogenerated addons API][addons-api].
+
```

Comment:
This looks a little bit weird when placed here. I think it might look better if we move this section to the beginning (before `Writing Addons`).

Reply:
The idea is to point to the API reference before demonstrating YAMLs that are immediately follow in the next "Enabling Addons" section.